qiskit/__init__.py

```diff
@@ -14,12 +14,25 @@
 
 """Main Qiskit public functionality."""
 
+from importlib.metadata import version, PackageNotFoundError
```

Comment:
Technically `importlib.metadata` was provisional in Python 3.8 and 3.9, though if what we need works in them, I think we're fine to use it without the backport package.

Reply:
Yeah it was provisional, but the api we're using here is unchanged between 3.8 and 3.10. I actually don't know how much any of the interfaces really changed during it's "provisional" period, the API in `importlib.metadata` isn't that large.

Reply:
Yeah, I think we're fine here.